easylinks/src/main/java/com/google/step/servlets/EasyLinkServlet.java

```diff
@@ -0,0 +1,40 @@
+package com.google.step.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responses to redirect the easy link to another website*/
+@WebServlet("/easy-link")
+public class EasyLinkServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String[] easyLink = ServletHelper.getParameterWithDefault(request, "easy-link", "").split("/");
+
+    switch (easyLink[0])
+    {
+      case EMPTY_LINK:
+        response.sendRedirect(HOME_PAGE);
+        break;
+      case PROVIDED_LINK:
```

Comment:
I guess you need something like easyLink[0][0] == "~" to check provided link, for example if the user types `~rmp`, easyLink will be `["~rmp"]`, and you need to get the first letter in the first word